site/src/examples/slider/CustomStep.tsx

```diff
@@ -6,19 +6,25 @@ export const CustomStep = (): ReactElement => (
   <StackLayout gap={10} style={{ width: "400px" }}>
     <FormField>
       <FormFieldLabel>Step: 1 (default)</FormFieldLabel>
-      <Slider min={-1} max={1} marks="all" />
+      <Slider min={-1} max={1} />
     </FormField>
     <FormField>
       <FormFieldLabel>Step: 0.2</FormFieldLabel>
-      <Slider min={-1} max={1} step={0.2} marks="all" />
```

Comment:
I get a strange rounding issue on 0.6 an -0.6 on this example, see the tooltip, it will have several decimal places